From 49797e8ae2753bacc74b86cd28be250ff93a62cc Mon Sep 17 00:00:00 2001
From: Stefano Facchini <stefano.facchini@gmail.com>
Date: Wed, 05 Sep 2012 17:34:19 +0000
Subject: printers: Add missing definition of HAVE_CUPS_1_6

Commit 1532d7ef forgot to actually define HAVE_CUPS_1_6
when building against CUPS 1.6

https://bugzilla.gnome.org/show_bug.cgi?id=683441
---
(limited to 'panels/printers/pp-maintenance-command.c')

diff --git a/panels/printers/pp-maintenance-command.c b/panels/printers/pp-maintenance-command.c
index e2a831b..0966945 100644
--- a/panels/printers/pp-maintenance-command.c
+++ b/panels/printers/pp-maintenance-command.c
@@ -25,6 +25,10 @@
 
 #include "pp-utils.h"
 
+#if (CUPS_VERSION_MAJOR > 1) || (CUPS_VERSION_MINOR > 5)
+#define HAVE_CUPS_1_6 1
+#endif
+
 #ifndef HAVE_CUPS_1_6
 #define ippGetCount(attr)     attr->num_values
 #define ippGetValueTag(attr)  attr->value_tag
--
cgit v0.9.0.2
